core/field_checkbox.js

```diff
@@ -127,11 +145,20 @@ Blockly.FieldCheckbox.prototype.initView = function() {
   this.textElement_.setAttribute('y', Blockly.FieldCheckbox.CHECK_Y_OFFSET);
   Blockly.utils.dom.addClass(this.textElement_, 'blocklyCheckbox');
 
-  var textNode = document.createTextNode(Blockly.FieldCheckbox.CHECK_CHAR);
-  this.textElement_.appendChild(textNode);
+  this.textContent_.nodeValue =
+      this.checkChar_ || Blockly.FieldCheckbox.CHECK_CHAR;
   this.textElement_.style.display = this.value_ ? 'block' : 'none';
 };
 
+/**
+ * Set the character used for the check mark.
+ * @param {string} character The character to use for the checkmark.
+ */
+Blockly.FieldCheckbox.prototype.setCheckCharacter = function(character) {
+  this.checkChar_ = character;
+  this.textContent_.nodeValue = character;
```

Comment:
Only if the field is initialized will ``this.textContent_`` have a value, do we need to check for that here? is it possible to call this before ``init``.

Reply:
Oh gosh golly, thanks for the catch! Fixed and added a test.